src/SonarScanner.MSBuild.Shim/Interfaces/ISonarScanner.cs

```diff
@@ -30,6 +30,6 @@ namespace SonarScanner.MSBuild.Shim
     /// file from them, then executes the Java sonar-scanner</remarks>
     public interface ISonarScanner
     {
-        ProjectInfoAnalysisResult Execute(AnalysisConfig config, IEnumerable<string> userCmdLineArguments);
+        bool Execute(AnalysisConfig config, IEnumerable<string> userCmdLineArguments, string propertiesFilePath);
```

Comment:
Is the interface still responsible for generating the properties file, or has it been done already i.e. is the new parameter telling the interface whether the properties file is, or where it should be created?
Either way, it's probably worth updating the doc comment for the interface.

Reply:
Ah yes you're right, we are doing it separately now, i'll update the doc.